src/services/profiles/types.ts

```diff
@@ -0,0 +1,107 @@
+import { ObjectId } from '../types';
+
+export interface Profile {
+  id?: ObjectId;
+  addressLine1?: string;
+  addressLine2?: string;
+  country?: string;
+  region?: string;
+  city?: string;
+  postalCode?: string;
+  birthday?: string;
+  /**
+   * See ISO 5218
+   */
+  gender?: number;
+  length?: number;
+  weight?: number;
+  afHistory?: boolean;
+  comorbidities?: Array<Comorbidities>;
+  physician?: string;
+  smoker?: boolean;
+  activity?: ProfileActivity;
+  impediments?: Array<Impediments>;
+  medication?: Array<Medication>;
+  groups?: Array<Group>;
+  customFields?: Record<string, string>;
+  fibricheckInfo?: string;
+  creationTimestamp?: Date;
+  updateTimestamp?: Date;
+}
+
+export type ProfileCreation = Pick<
```

Comment:
When I call sdk.profiles.create with an empty object. I get `required_fields: [ 'birthday', 'country', 'gender', 'id' ]`

Reply:
erm so what exactly are you expecting? I'm not sure what are the required fields..

Reply:
Check the swagger docs. https://developers.extrahorizon.io/swagger-ui/?url=https://developers.extrahorizon.io/services/profiles-service/1.1.3/openapi.yaml#/Profile/post_ 
Only `region` is not required according to the swagger

Reply:
yes you're right, I missed that. So I've changed it, have a look and let me know. Thanks!